lib/jekyll/commands/new.rb

```diff
@@ -88,8 +88,14 @@ def gemfile_contents
             group :jekyll_plugins do
               gem "jekyll-feed", "~> 0.6"
             end
-            # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-            gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+            # Windows and JRuby does not include zoneinfo files, so bundle the tzinfo-data gem
+            # and associated library.
+            install_if -> { RUBY_PLATFORM =~ %r!mingw|mswin|java! } do
```

Comment:
Any reason you don't use the same check as in the wdm gem? `Gem.win_platform?`

Reply:
I wanted to test for both Windows and JRuby with one operation.